.eslintrc

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "eslint-config-fliplet"
```

Comment:
how does this connect to the actual file?

Reply:
@squallstar 
Threw the npm, we use this [repository](https://github.com/YaroslavOvdii/FlipletEsLint).
And this [line](https://github.com/Fliplet/fliplet-widget-slider/pull/27/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R7) in the package.json to connect to the index.js in the repository.